active/0019-plugins.md

```diff
@@ -0,0 +1,126 @@
+# EMQ X 5.0 plugins
+
+## Change log
+
+* 2021-11: Guowei Li
+* 2021-12-07: @zmstone move it to GitHub
+
+## Abstract
+
+This EIP documents the implementation proposal EMQ X 5.0 plugins.
+
+## Background
+
+Prior to EMQ X 5.0, most of the features are implemented as plugins. A plugin is an Erlang application which registers a set of call back APIs at certain pre-defined hook points.
+
+The applications are configured and loaded separately after the node is booted.
+
+In 5.0, although most of the features such as authentication, authorisation, and rule-engine are still implemented by registering hooks, the management of the features are no longer as the old plugins. 
```

Comment:
For my own clarification: "management" here means managing the repos of the plugins themselves?  Or does it mean the configuration?

Reply:
it means how the features are enabled/disabled or configured.